enclave_wrangler/new_enclave_api.py

```diff
@@ -0,0 +1,649 @@
+#           "description": "",
+#           "baseType": "OntologyObject"
+#         },
+#         "version": {
+#           "description": "",
+#           "baseType": "OntologyObject"
+#         },
+#         "include_mapped": {
+#           "description": "If true, then these expression items will match on the selected OMOP Concepts, and all of the Non-Standard OMOP Concepts that map to them. If Include Descendants is also true, then this option will also include all OMOP Concepts that map to the included descendants. Setting this to true enables you to include non-standard Concepts in your Concept Set. Mapping is the process to transform one Concept into a Standard one. Read more: ohdsi.github.io/TheBookOfOhdsi/Cohorts.html#conceptSets and https://www.ohdsi.org/web/wiki/doku.php?id=documentation:vocabulary:mapping",
+#           "baseType": "Boolean"
+#         }
+#       }
+#     },
+#  B. How about this one?
+#      - attn: 'concepts' (Array<OntologyObject>). are these new or previously uploaded ones?
+#     {
+#       "apiName": "add-selected-concepts-as-omop-version-expressions",
+#       "description": "",
+#       "rid": "ri.actions.main.action-type.d1ad39f8-a303-4f46-8f46-bd48c5362915",
+#       "parameters": {
+#         "concepts": {
+#           "description": "",
+#           "baseType": "Array<OntologyObject>"
+#         },
+#         "includeMapped": {
+#           "description": "If true, then these expression items will match on the selected OMOP Concepts, and all of the Non-Standard OMOP Concepts that map to them. If Include Descendants is also true, then this option will also include all OMOP Concepts that map to the included descendants. Setting this to true enables you to include non-standard Concepts in your Concept Set. Mapping is the process to transform one Concept into a Standard one. Read more: ohdsi.github.io/TheBookOfOhdsi/Cohorts.html#conceptSets and https://www.ohdsi.org/web/wiki/doku.php?id=documentation:vocabulary:mapping",
+#           "baseType": "Boolean"
+#         },
+#         "includeDescendants": {
+#           "description": "If true, then these expression items will match on the selected OMOP Concepts, and all of their descendants.",
+#           "baseType": "Boolean"
+#         },
+#         "version": {
+#           "description": "",
+#           "baseType": "OntologyObject"
+#         },
+#         "isExcluded": {
+#           "description": "If true, then any concepts matched will be added to the expression as exclusion rather than inclusion criteria. Exclusion criteria take precedence over inclusion criteria, in cases when a single OMOP Concept is affected by more than one entry in the OMOP Concept Set Expression.",
+#           "baseType": "Boolean"
+#         },
+#         "optional-annotation": {
+#           "description": "What are you trying to accomplish? Reason?",
+#           "baseType": "String"
+#         }
+#       }
+#     },
+def upload_concept(
```

Comment:
@Sigfried `upload_concept()`: `# TODO: How to know which concept set and version to add it to?`
I looked at the endpoint you suggested, along with the one we're currently using, and another one. Together, the relevant endpoints to upload new concept version items seem to be:
1. `set-omop-concept-set-version-item`
2. `edit-omop-concept-set-version-item`
3. `add-selected-concepts-as-omop-version-expressions`

However, it's still not clear to me how to connect concept version items with their cset versions. I don't see any param at any of these 3 endpoints for that. It could be that (a) there is a param to do it, but I just don't understand which param to use, or (b) maybe there's another endpoint for version items to do this, or (c) perhaps we should look at cset version endpoints (other than the one we're currently using to upload a new version, which is `create-new-draft-omop-concept-set-version`). Maybe there's a way to to reference concepts and add them that way.